libraries/protocol/include/steem/protocol/smt_operations.hpp

```diff
@@ -13,8 +13,13 @@
 #define SMT_MIN_SATURATION_STEEM_UNITS 1000
 #define SMT_MIN_SOFT_CAP_STEEM_UNITS   1000
 
-namespace steem { namespace protocol {
+#define SMT_DESTINATION_FROM          account_name_type( "$from" )
+#define SMT_DESTINATION_FROM_VESTING  account_name_type( "$from.vesting" )
+#define SMT_DESTINATION_MARKET_MAKER  account_name_type( "$market_maker" )
```

Comment:
I don't see that market maker is used anywhere. While we are not launching with the market maker enabled, ICOs should still be able to route tokens (both STEEM and SMTs) to the market maker for future use.